ui/edit_profile.html

```diff
@@ -48,7 +48,8 @@
                 </tr>
                 <tr>
                     <td><label> <h4><b><font color="white">Gender:</font></b> </label></td></h4>
-                    <td><h4><input type="text" name="gender" value="{{pcuser.gender}}"></h4></td>
+                    <td><h4><select name="gender"><option value="" selected disabled>Select</option>
```

Comment:
@prafful002 @Nerdylicious   do we indicate required and non-required fields? or are all fields required?